src/app/donation-start/donation-start.component.html

```diff
@@ -1,6 +1,5 @@
-<h1>Donating to campaign ID {{ campaignId }}!</h1>
-
-<form (ngSubmit)="submit()" [formGroup]="donationForm">
+<form class="formPadding" (ngSubmit)="submit()" [formGroup]="donationForm">
```

Comment:
If the component only has one `form`, why not just target `form` in the SCSS? Angular will isolate styling's scope to the component for you.